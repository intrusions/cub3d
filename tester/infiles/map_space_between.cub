NO ./texture/no.xpm
SO ./texture/so.xpm
WE ./texture/we.xpm
EA ./texture/ea.xpm

F 220,100,0
C 225,30,0

11111111111111111111
10000000000000000001
10011000000000000001
10011000000000000001
10000000000011000001
10000000000011000001
10000000000000000001
10000000000000000001
11111111111000000001
10000000001000000001
10000000001000000001

10000000001000000001
10000000001000000001
10000000001000000001
10000000001000000001
10000000000000000001
10000000000000000001
11111111111000000001
100N0000000000000001
11111111111111111111