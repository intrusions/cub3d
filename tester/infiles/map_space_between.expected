Loading scene...
Loading map...
Error
	map_space_between.cub: Map: is open
Quitting...
